configs/components/cpp-pcp-client.rb

```diff
@@ -59,7 +59,10 @@
 
     cmake = 'C:/ProgramData/chocolatey/bin/cmake.exe -G "MinGW Makefiles"'
     toolchain = "-DCMAKE_TOOLCHAIN_FILE=#{settings[:tools_root]}/pl-build-toolchain.cmake"
-  elsif platform.name =~ /el-[67]|redhatfips-7|sles-1[12]|ubuntu-18.04-amd64/
+  elsif platform.name == 'sles-11-x86_64'
+    cmake = 'env LD_LIBRARY_PATH=/opt/pl-build-tools/lib64 /opt/pl-build-tools/bin/cmake'
```

Comment:
Wondering if the `LD_LIBRARY_PATH` is needed because the `lib64` directory is already in the RPATH: 
```
chromatic-guilt:~ # readelf -a /opt/pl-build-tools/bin/cmake | grep RUNPATH
 0x000000000000001d (RUNPATH)            Library runpath: [/opt/pl-build-tools/lib:/opt/pl-build-tools/lib64]
```

And the linker can resolve dependencies without it 

chromatic-guilt:~ # ldd /opt/pl-build-tools/bin/cmake
	linux-vdso.so.1 =>  (0x00007ffc8bbad000)
	libssl.so.0.9.8 => /usr/lib64/libssl.so.0.9.8 (0x00007f5a9430d000)
	libcrypto.so.0.9.8 => /usr/lib64/libcrypto.so.0.9.8 (0x00007f5a93f6c000)
	libdl.so.2 => /lib64/libdl.so.2 (0x00007f5a93d68000)
	librt.so.1 => /lib64/librt.so.1 (0x00007f5a93b5f000)
	libpthread.so.0 => /lib64/libpthread.so.0 (0x00007f5a93942000)
	libstdc++.so.6 => /opt/pl-build-tools/lib64/libstdc++.so.6 (0x00007f5a935b6000)
	libm.so.6 => /lib64/libm.so.6 (0x00007f5a9333b000)
	libgcc_s.so.1 => /opt/pl-build-tools/lib64/libgcc_s.so.1 (0x00007f5a93123000)
	libc.so.6 => /lib64/libc.so.6 (0x00007f5a92da7000)
	libz.so.1 => /lib64/libz.so.1 (0x00007f5a92b91000)
	/lib64/ld-linux-x86-64.so.2 (0x00007f5a94565000)
```

Reply:
Likely not needed (but certainly needs confirmation) since I've eliminated the two other GCCs in the original example so the C++ runtimes shouldn't be battling. This is a remnant of where I started poking at it.